storage-resize-images/functions/lib/resize-image.js

```diff
@@ -29,24 +29,32 @@ function resize(file, size) {
 }
 exports.resize = resize;
 function convertType(buffer, format) {
+    let outputOptions = { jpeg: {}, png: {}, webp: {}, tiff: {} };
```

Comment:
Do we need `tif` and `jpg` as alternative spellings? Or documentation to explain that if a user includes `tif` for example that they would need to use `tiff` for output options?